### PR TITLE
Restore search box border; Closes #115;

### LINKED ISSUE #115: Search bar outline is invisible
**Describe the bug**

Search bar outline is invisible

**Steps to reproduce the bug**

N/A

**Expected behavior**

Search bar would have an outline to discern it from everything else

**Actual behavior**

Search box doesn't look like a search box

**Screenshots**

https://user-images.githubusercontent.com/51363864/60203697-8aa08900-9890-11e9-8399-e4f465b34d92.png

**Version Info**

Gallery version (found on Settings page): 

Windows 10 version:
- [ ] Insider Build (xxxxx)
- [x] May 2019 Update (18362)
- [ ] October 2018 Update (17763)
- [ ] April 2018 Update (17134)
- [ ] Fall Creators Update (16299)
- [ ] Creators Update (15063)
- [ ] Anniversary Update (14393)

Device form factor:
- [x] Desktop
- [ ] Mobile
- [ ] Xbox
- [ ] Surface Hub
- [ ] IoT

**Additional context**

https://github.com/microsoft/microsoft-ui-xaml/issues/947#issuecomment-507767170

### PR DESCRIPTION
**Pull request by chingucoding**
Restored the border for the search box to make it stand out more as a search box.
## Description

Removed the styling of the AutoSuggestBoxStyle in [App.xaml](https://github.com/microsoft/Xaml-Controls-Gallery/blob/master/XamlControlsGallery/App.xaml) and removed the requested theme in  [NavigationRootPage.xaml](https://github.com/microsoft/Xaml-Controls-Gallery/blob/master/XamlControlsGallery/Navigation/NavigationRootPage.xaml)
## Motivation and Context

Fixes #115. Also see [this comment](https://github.com/microsoft/microsoft-ui-xaml/issues/947#issuecomment-507767170).
## How Has This Been Tested?

Tested by starting application manually.
## Screenshots (if appropriate):
"New" search bar in light theme:
image

"New" search bar in dark theme:
image

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: XamlControlsGallery/App.xaml [hunk 1]
```diff
@@ -393,7 +393,6 @@
             <Style x:Key="AutoSuggestBoxStyle" TargetType="AutoSuggestBox">
```
Comment (YuliKl) [inline XamlControlsGallery/App.xaml]: I think it's fine to delete this style completely, it's really not accomplishing very much at this point.

File: XamlControlsGallery/Navigation/NavigationRootPage.xaml [hunk 2]
```diff
@@ -84,7 +84,6 @@
                     PlaceholderText="Search"
```
Comment (YuliKl) [inline XamlControlsGallery/Navigation/NavigationRootPage.xaml]: In addition to removing the RequestedTheme, I would also delete the Style setter. I think it's fine for this ASB to just have the default look and feel.

File: XamlControlsGallery/App.xaml [hunk 3]
```diff
@@ -393,7 +393,6 @@
...
                 <Setter Property="VerticalAlignment" Value="Top" />
                 <Setter Property="IsTabStop" Value="False" />
-                <Setter Property="TextBoxStyle" Value="{StaticResource CustomAutoSuggestBoxTextBoxStyle}" />
```
Comment (YuliKl) [inline XamlControlsGallery/App.xaml]: CustomAutoSuggestBoxTextBoxStyle is now not being used by anything so I think it's safe to delete out of App.xaml.

File: XamlControlsGallery/App.xaml (hunk 1 above)
Comment (chingucoding) [reply XamlControlsGallery/App.xaml]: Removed the style in the latest commit.

File: XamlControlsGallery/Navigation/NavigationRootPage.xaml (hunk 2 above)
Comment (chingucoding) [reply XamlControlsGallery/Navigation/NavigationRootPage.xaml]: Removed it in the latest commit.

File: XamlControlsGallery/App.xaml (hunk 3 above)
Comment (chingucoding) [reply XamlControlsGallery/App.xaml]: Removed that too in the latest commit. Now all "dead" styling code should be removed ^^

**Pull request merged**